static/js/server_events_dispatch.js

```diff
@@ -329,6 +329,8 @@ export function dispatch_normal_event(event) {
                     bot_data.add(event.bot);
                     settings_bots.render_bots();
                     settings_users.update_bot_data(event.bot.user_id);
```

Comment:
We can remove this. Since this calls `render_item` which replaces the existing row with new html and there is no existing row in case of adding a new bot.

Reply:
Done.